barretenberg/cpp/src/barretenberg/vm2/constraining/recursion/recursive_verifier.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "barretenberg/stdlib/plonk_recursion/aggregation_state/aggregation_state.hpp"
+#include "barretenberg/sumcheck/sumcheck.hpp"
+#include "barretenberg/vm2/constraining/recursion/recursive_flavor.hpp"
+
+namespace bb::avm2 {
+
+template <typename Flavor> class AvmRecursiveVerifier_ {
```

Comment:
Why do we need to call it `AvmRecursiveVerifier_` with `_`?

Reply:
we've historically done this to facilitate aliases: `using AvmRecursiveVerifier = AvmRecursiveVerifier_<SpecificParam>;`